<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SOLVESLAE</class>
 <widget class="QDialog" name="SOLVESLAE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="main_layout">
     <item>
      <layout class="QVBoxLayout" name="right_part_layout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Matrix of system</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTableWidget" name="matrix_table_widget"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="buttons_layout">
         <item>
          <layout class="QHBoxLayout" name="set_dim_layout">
           <item>
            <widget class="QPushButton" name="set_dim_button">
             <property name="text">
              <string>Set dimension</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="solve_button">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>41</height>
            </size>
           </property>
           <property name="text">
            <string>Solve</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="plot_solution_button">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>41</height>
            </size>
           </property>
           <property name="text">
            <string>Plot solution</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="done_bitton">
           <property name="minimumSize">
            <size>
             <width>371</width>
             <height>41</height>
            </size>
           </property>
           <property name="text">
            <string>Done</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="left_part_layout">
       <item>
        <widget class="QGroupBox" name="b_vec_group_box">
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>B - column</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QTableWidget" name="b_column_table_widget">
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="res_vec_group_box">
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Result column</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QTableWidget" name="result_column_table_widget">
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
